plan2nix/Plan2Nix.hs

```diff
@@ -206,17 +206,15 @@ value2plan plan = Plan { packages, extras, compilerVersion, compilerPackages }
       }
     _ -> Nothing
 
-  extras = fmap Just $ filterInstallPlan $ \pkg -> case ( pkg ^. key "type" . _String
-                                                        , pkg ^. key "style" . _String
-                                                        , pkg ^. key "pkg-src" . key "type" . _String
-                                                        , pkg ^. key "pkg-src" . _Object) of
-    (_, "local", "local", _) -> Just $ Package
+  extras = fmap Just $ filterInstallPlan $ \pkg -> case ( pkg ^. key "style" . _String
+                                                        , pkg ^. key "pkg-src" . key "type" . _String) of
+    ("local", "local") -> Just $ Package
```

Comment:
I'm actually not sure why we care about the style at all. Wouldn't pkg-src be the only important thing?

Reply:
It seems like `style` is now directly derived from `pkg-src`, from my understanding of the Cabal code.